src/components/explore/ContentGrouping.tsx

```diff
@@ -1,19 +1,24 @@
 "use client";
 
-import { createContentSlug, createSlug, GroupedData } from "@/utils";
+import {
+  createContentSlug,
+  createSlug,
+  ExploreGroupedData,
+  NavigationList,
+} from "@/utils";
 import Link from "next/link";
-import { SetStateAction, useEffect, useRef } from "react";
+import { useEffect, useRef } from "react";
 import { twMerge } from "tailwind-merge";
 
 export interface IContentGrouping {
   currentGroup: string;
-  groupedData: GroupedData | never[];
+  navigationList: NavigationList[] | ExploreGroupedData[]; // Account for Explore Page and Transcript Page
   screen?: "mobile" | "desktop";
   className?: string;
 }
 const ContentGrouping = ({
```

Comment:
This naming doesn't seem very descriptive.
Its more or less Navigation by words right?
There are a lot of components with grouping and transcript names in them it can be hard to track

Reply:
Yh, it is